### PR TITLE
Supporting dynamic ports

### PR DESCRIPTION
**Pull request by bfosberry**
Not urgent <3

### CODE REVIEW

**Comment by bfosberry** [comment]
Having issues with this, hold off on merging

**Comment by bfosberry** [comment]
Ok, gtg

File: dockerutils.go [hunk 1]
```diff
@@ -132,23 +132,28 @@ func DockerPorts(expose []uint16, ports []string) (map[docker.Port]struct{}, map
...
 		if err != nil {
 			return nil, nil, err
 		}
-		dockerPort := docker.Port(fmt.Sprintf("%v/tcp", containerPort))
+		dockerPort := docker.Port(fmt.Sprintf("%s/tcp", containerPort))
 		m[dockerPort] = emptyStruct()
 		n[dockerPort] = []docker.PortBinding{
 			docker.PortBinding{
-				HostPort: fmt.Sprintf("%d", hostPort),
+				HostPort: fmt.Sprintf("%s", hostPort),
```
Comment (peter-edge) [inline dockerutils.go]: You don't need the fmt.sprintf anymore :)

Comment (peter-edge) [inline dockerutils.go]: I'm just going to merge this and then I'll fix this on master, thanks :)

**Pull request merged**